client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -70,26 +69,11 @@ class CustomContentTypes extends Component {
 
 	renderHeader() {
```

Comment:
I don't think this method is necessary anymore - it can be a one-liner in the `render()` method.